Loenn/entities/arrowBubble.lua

```diff
@@ -0,0 +1,31 @@
+local arrowBubble = {}
+
+arrowBubble.name = "isaBag/arrowBubble"
+arrowBubble.depth = -100
+arrowBubble.placements = {
+    {
+        name = "Arrow Bubble (Left, IsaGrabBag)",
+        data = {
+            direction = left
```

Comment:
This doesn't load lol, looks like it's WIP though. I'd recommend making the `direction` field a non-editable dropdown with the directions. Looks like the code needs it to be lowercase which is annoying, I think there's a way to make it display separately though